week-4/Homework/mandatory/2-writers.js

```diff
@@ -22,29 +22,29 @@ let writers = [
     lastName: "Woolf",

```

Comment:
Also please try to follow the github process correctly when making a pull request :) Changes from weeks 4 and 5 should not be showing in this pull request.